webpack/assets/javascripts/react_app/components/common/forms/OrderableSelect/helpers.js

```diff
@@ -0,0 +1,123 @@
+import React, { useImperativeHandle, useRef } from 'react';
+import { DragSource, DropTarget } from 'react-dnd';
+import PropTypes from 'prop-types';
+import { set } from 'lodash';
+
+export const orderDragged = (inputArry, dragIndex, hoverIndex) => {
+  const dragedValue = inputArry[dragIndex];
```

Comment:
typo ? `inputArray`

Reply:
It wasn't - I like to avoid naming vars by type names. I agree it seems like it though.